gcloud/pubsub/_gax.py

```
@@ -0,0 +1,173 @@
+            :param topic_path: fully-qualfied path of the new topic, in format
+                               ``projects/<PROJECT>/topics/<TOPIC_NAME>``.
+
+            :rtype: dict
+            :returns: ``Topic`` resource returned from the API.
+            :raises: :exc:`gcloud.exceptions.Conflict` if the topic already 
+                     exists
+            """
+            try:
+                topic_pb = self._gax_api.create_topic(topic_path)
+            except GaxError:
+                raise Conflict(topic_path)
+            return {'name': topic_pb.name}
+
+        def topic_get(self, topic_path):
+            """API call:  retrieve a topic
+
+            See:
+            https://cloud.google.com/pubsub/reference/rest/v1/projects.topics/get
+
+            :type topic_path: string
+            :param topic_path: fully-qualfied path of the topic, in format
+                            ``projects/<PROJECT>/topics/<TOPIC_NAME>``.
+
+            :rtype: dict
+            :returns: ``Topic`` resource returned from the API.
+            :raises: :exc:`gcloud.exceptions.NotFound` if the topic does not
+                     exist
+            """
+            try:
+                topic_pb = self._gax_api.get_topic(topic_path)
+            except GaxError:
+                raise NotFound(topic_path)
+            return {'name': topic_pb.name}
+
+        def topic_delete(self, topic_path):
+            """API call:  delete a topic
+
+            See:
+            https://cloud.google.com/pubsub/reference/rest/v1/projects.topics/create
+
+            :type topic_path: string
+            :param topic_path: fully-qualfied path of the new topic, in format
+                               ``projects/<PROJECT>/topics/<TOPIC_NAME>``.
+
+            :rtype: dict
+            :returns: ``Topic`` resource returned from the API.
+            """
+            try:
+                self._gax_api.delete_topic(topic_path)
+            except GaxError:
+                pass
```

8893362 makes `topic_delete` raise a `NotFound` instead of ignoring the `GaxError`.